src/ontology/hp-edit.owl

```diff
@@ -33752,7 +33721,6 @@ AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#has
 AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#id> <http://purl.obolibrary.org/obo/HP_0001626> "HP:0001626"^^xsd:string)
 AnnotationAssertion(rdfs:comment <http://purl.obolibrary.org/obo/HP_0001626> "The cardiovascular system consists of the heart, vasculature, and the lymphatic system."^^xsd:string)
 AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/HP_0001626> "Abnormality of the cardiovascular system"^^xsd:string)
-EquivalentClasses(<http://purl.obolibrary.org/obo/HP_0001626> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000051> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/PATO_0000051> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002314> <http://purl.obolibrary.org/obo/UBERON_0004535>) ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002573> <http://purl.obolibrary.org/obo/PATO_0000460>))))
```

Comment:
do we have a replacement for this? @nicolevasilevsky @matentzn

Reply:
Yes, will have. Not important now; the next step in the patternisation involves fully defining all these top level classes.